data/themes/default/style.css

```diff
@@ -278,33 +283,33 @@ QScrollBar::add-page:vertical:pressed, QScrollBar::sub-page:vertical:pressed {
 /* scrollbar: handles (sliders) */
 
 QScrollBar::handle:horizontal {
-	background: #3f4750;
+	background: rgb(183, 217, 195);
 	border: none;
 	border-radius: 4px;
 	min-width: 24px;
 }
 
 QScrollBar::handle:horizontal:hover {
-	background: #525e69;
+	background: #cff3e4;
```

Comment:
The current implementation uses a mix of `rgb(x, y, z)` and definitions with hashes a la `#caffee`. It think it should be unified in the `QScrollBar` section.

Reply:
I have removed the `rgb(x, y, z)` from that place so should be consistent now.

Reply:
fixed in https://github.com/LMMS/lmms/pull/7202/commits/da455e966753e575180a9fbb92528affe1489695